app/views/layouts/application.html.erb

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag "application.tailwind", "inter-font", "data-turbo-track": "reload" %>
```

Comment:
Let's revert this change, it breaks the styles for login form.

Run `$ rails tailwindcss:watch` locally to have this build

Reply:
Ok, I will revert it.